code/datums/elements/food/processable.dm

```diff
@@ -0,0 +1,33 @@
+// If an item has the processable item, it can be processed into another item with a specific tool. This adds generic behavior for those actions to make it easier to set-up generically.
+/datum/element/processable
+	element_flags = ELEMENT_BESPOKE
+	id_arg_index = 2
+	///The type of atom this creates when the processing recipe is used.
+	var/atom/result_atom_type
```

Comment:
```suggestion
	var/result_atom_type
```
Don't use this type, which is what we use to tell the compiler this is an instantiated entity if not `null`